docs/kit/hardware/motor-board.md

```diff
@@ -0,0 +1,34 @@
+# Motor Board
+
+![Motor Board LEDs](https://raw.githubusercontent.com/srobo/motor-v4-hw/master/test/led_diagram.jpg)
+
+!!! note
+
+    The schematic has motors 0 and 1 reversed from the silkscreen and code.
```

Comment:
Should we file an issue to fix the schematic?

Reply:
This would be something that @WillB97 or @8BitJosh will be able to tell you. I can't think of a reason why we couldn't.

Reply:
This is non-trivial to fix.

Reply:
https://github.com/srobo/motor-v4-hw/issues/6 exists, which was what I was after here (i.e: that this wasn't the only record of this outstanding work). Appreciate that actually fixing it may be non-trivial.